src/controllers/post.js

```diff
@@ -1,15 +1,29 @@
-import { sendDataResponse } from '../utils/responses.js'
+import { sendDataResponse, sendMessageResponse } from '../utils/responses.js'
 import { PrismaClient } from '@prisma/client'
+import jwt from 'jsonwebtoken'
+import { JWT_SECRET } from '../utils/config.js'
 const prisma = new PrismaClient()
 
 export const create = async (req, res) => {
   const { content } = req.body
+  const token = req.headers.authorization.split(' ')
 
   if (!content) {
-    return sendDataResponse(res, 400, { content: 'Must provide content' })
+    return sendMessageResponse(res, 400, 'Must provide content')
   }
 
-  return sendDataResponse(res, 201, { post: { id: 1, content } })
+  try {
+    const user = jwt.verify(token[1], JWT_SECRET)
```

Comment:
Token authentication is already happening in the app.

`.verify` is used to verify a JWT, which already happens in the auth middleware. When the auth middleware verifies the request, it will attach the existing user to the request object (`req`) - see the `validateAuthentication` function in `src/middleware/auth.js`

You should use the `user` that is already attached to the request instead of working with a token